tests/test_scylla.c

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright (C) 2019 BiiLabs Co., Ltd. and Contributors
+ * All Rights Reserved.
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the MIT license. A copy of the license can be found in the file
+ * "LICENSE" at the root of this distribution.
+ */
+
+#include "storage/scylla_api.h"
+#include "test_define.h"
+
+static char* host;
+
+void test_get_column_from_edgeTable(CassSession* session) {
+  char expected_result[][FLEX_TRIT_SIZE_243] = {
+      {BUNDLE_1},       // address_2
+      {BUNDLE_2},       // address_2
+      {BUNDLE_3},       // address_2
+      {BUNDLE_1},       // address_1
+      {TRANSACTION_4},  // approvees T7
+      {TRANSACTION_3},  // approvees T7
+      {TRANSACTION_6},  // approvees T7
+      {TRANSACTION_5}   // approvees T7
+  };
+  hash243_queue_t res_hash = NULL;
+  hash243_queue_entry_t* iter243 = NULL;
+  get_column_from_edgeTable(session, &res_hash, (cass_byte_t*)ADDRESS_2, "bundle");
+  get_column_from_edgeTable(session, &res_hash, (cass_byte_t*)ADDRESS_1, "bundle");
+  get_column_from_edgeTable(session, &res_hash, (cass_byte_t*)TRANSACTION_7, "hash");
+  size_t idx = 0;
+  CDL_FOREACH(res_hash, iter243) {
+    TEST_ASSERT_FALSE(
```

Comment:
is it a unity MACRO? If so, maybe use TEST_ASSERT_EQUAL_MEMORY instead

Reply:
It seems better to use TEST_ASSERT_EQUAL_MEMORY!